ledger/participant-local-store/src/test/lib/com/daml/platform/localstore/IdentityProviderConfigStoreTests.scala

```diff
@@ -264,6 +265,47 @@ trait IdentityProviderConfigStoreTests extends IdentityProviderConfigStoreSpecBa
       }
     }
 
+    "allows to update existing identity provider config's audience attribute" in {
```

Comment:
I think this needs to be extended a bit. you only setup a single IDP here and are testing that. I think we need some auth tests that check what happens if we have multiple IDPs some of which require (different) audiences and some don't set a custom audience.

Reply:
yes, I will add a test like that